### DISCUSSION TITLE
Fallback jpeg not working when IIIF image is absent

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#917] [Get Help]
Our Italian Paintings project uses existing IIIF images for all of the catalogued object images, like this one:

https://italian-paintings-live.netlify.app/catalogue/volume-1/the-florentine-school/9/

I believe the site is supposed to use the fallback jpeg if ever this IIIF image is not available, but that is not happening. Occasionally, if the museum's Visual Resources department updates the IIIF image, it disappears from our site, like this one, although it won't be an example for long because I will update the IIIF url and canvas ID as soon as I am able:

https://italian-paintings-live.netlify.app/catalogue/volume-1/the-florentine-school/14/

@Erin-Cecele , is this a bug? Shouldn't the site use the fallback jpeg if the IIIF info is suddenly incorrect? I see there are references to the fallback jpeg in the v0 documentation, but not in the v1 documentation. Thank you!

**Comment by Erin-Cecele** [comment]
Hi @zsofiaj, This is not a bug. You are correct that Quire uses a fallback image in place of a IIIF image in the PDF and e-book in both v0 and v1.  However, Quire is not designed use a fallback on the website when it encounters a broken or missing IIIF image. While I'm not sure if it's even feasible, it's a nice idea so I'm tagging this as an enhancement for us to look at in the future.

**Reply by zsofiaj** [reply]
Thank you for clarifying, @Erin-Cecele ! That makes sense. I read the word "fallback" to mean the jpeg would be used if the IIIF image is not available for some reason.
